src/pages/login/login.js

```diff
@@ -49,6 +49,26 @@ const Login = () => {
       .matches(formRegExp.password, errorMessages[language].value.password)
       .required(' ')
   });
+  const auth = () => {
+    const _onInit = (auth2) => {
```

Comment:
We do not use `_`-starting naming, rename this method to make it more clear what it does, also use proper naming for parameters

Reply:
Fixed